cmd/root.go

```diff
@@ -85,7 +80,7 @@ var rootCmd = &cobra.Command{
 func Execute() {
 	defer func() {
 		if err := recover(); err != nil {
-			pterm.Error.Printfln(
+			_, _ = tui.Error.Printfln(
```

Comment:
Does this need to be assigned?
```suggestion
			tui.Error.Printfln(
```

Reply:
was getting a weird lint error if I didnt 👯

Reply:
this could be addressed by not returning the result in the prefix printer. I'm pretty sure the `fmt` package has exceptions for error/assignment handling.

Reply:
Pushed changes